pkg/cnb/cnb_metadata.go

```diff
@@ -12,7 +12,10 @@ import (
 	"github.com/pivotal/kpack/pkg/registry"
 )
 
-const BuilderMetadataLabel = "io.buildpacks.builder.metadata"
+const (
+	BuilderMetadataLabel = "io.buildpacks.builder.metadata"
+	StackLabel           = "io.buildpacks.stack.id"
```

Comment:
We should probably import this labels from the lifecycle so that we don't run into problems if it changes. I don't think it really will, but it seems like a safer way to use it

Reply:
The lifecycle does not have a label for builder metadata tho but I we should use constants from the lifecycle where we can